Source/KSCrash/Source/KSCrash/Recording/Sentry/BSG_KSCrashSentry_User.c

```diff
@@ -101,11 +101,7 @@ void bsg_kscrashsentry_reportUserException(const char *name, const char *reason,
         int callstackCount = 100;
         uintptr_t callstack[callstackCount];
 
-        if (localContext->suspendThreadsForUserReported) {
```

Comment:
Why are we not honouring `suspendThreadsForUserReported`?

Reply:
This is always set to `true` and is an internal property of KSCrash which isn't exposed. For now I've reinstated the checks here but it would make sense to fully remove `suspendThreadsForUserReported` at a later date as it's not used.